### DISCUSSION TITLE
Bezier Curves for smoother line

### DISCUSSION THREAD

**Discussion by erikwii** [szimek/signature_pad#588] [Ideas]
Hi,
I know it's not a new idea but I think it's worth discussing
Can you guys add an option to make the lines smoother like in "Procreate" with Bezier Curves or something?

I've seen other [libraries](https://www.jqueryscript.net/demo/Smooth-Signature-Pad-Plugin-with-jQuery-Html5-Canvas/examples/) that use this technique, but not as well as Procreate.
Thanks in advance.

**Comment by UziTech** [comment]
PRs are always appreciated 😁👍
